README.md

```diff
@@ -523,6 +523,55 @@ query('getHeroForEpisode', {
 })
 ```
 
+If you are using a discriminated union pattern, then you can use the `onUnion` helper, which will automatically generate the union type for you:
+
+```graphql
```

Comment:
I think the query will be like

```graphql
query getHeroForEpisode {
  hero {
    id
    ... on Droid {
      kind              # <-- `kind` will be added
      primaryFunction
    }
    ... on Human {
      kind              # <-- `kind` will be added
      height
    }
  }
}
```

Reply:
whoops forgot about it I think, fixed now